server/chat-commands/core.ts

```diff
@@ -1437,7 +1484,7 @@ exports.commands = {
 
 		let commaIndex = target.indexOf(',');
 		let targetName = target;
-		let targetRegistered = false;
+		let targetRegistered: any = false;
```

Comment:
Again, why are you using `any` here? There's no real point in migrating a file to TypeScript if you're throwing out `any` or `AnyObject` at any slightly difficult type.

Your goal is not to shut up the compiler; it's to look at it's concerns and fix the issue.